src/solvers/strings/string_dependencies.h

```diff
@@ -0,0 +1,198 @@
+/*******************************************************************\
+
+Module: String solver
+
+Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+#ifndef CPROVER_SOLVERS_STRINGS_STRING_DEPENDENCIES_H
```

Comment:
doxygen `\file` is missing.

Reply:
@peterschrammel Is that really useful? The class is already documented.

Reply:
I would only repeat the description.

Reply:
@allredj, please check whether it generates an entry next to the file here: http://cprover.diffblue.com/files.html

Reply:
Indeed there's no description on the file view. I added a file documentation for this file.